backend/check_persona_system_prompts.py

```
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)